cime/config/e3sm/machines/config_machines.xml

```diff
@@ -418,7 +418,7 @@
       </modules>
 
       <modules mpilib="impi">
-        <command name="swap">cray-mpich impi/2018.up2</command>
+        <command name="swap">cray-mpich impi/2019.up3</command>
```

Comment:
Please note that `impi/2019.up3` is hanging on many tests with a message
```
 0: [0] MPI startup(): I_MPI_OFI_LIBRARY variable has been removed from the product, its value is ignored
 0:
 0: [0] MPI startup(): I_MPI_OFI_LIBRARY environment variable is not supported.
 0: [0] MPI startup(): Similar variables:
 0:      I_MPI_OFI_LIBRARY_INTERNAL
```
Things are working better with `impi/2018.up3`.